python/pyspark/ml/feature.py

```diff
@@ -3466,6 +3466,72 @@ def selectedFeatures(self):
         return self._call_java("selectedFeatures")
 
 
+@inherit_doc
+class VectorSizeHint(JavaTransformer, HasInputCol, HasHandleInvalid, JavaMLReadable,
```

Comment:
You'll need to override handleInvalid, like in the Scala API, since it takes different values & has a different docstring.